src/showingpreviously/cinemas/odeon.py

```diff
@@ -0,0 +1,146 @@
+import json
+import re
+
+from selenium import webdriver
+from datetime import datetime, timedelta
+from typing import Tuple, Iterator
+import showingpreviously.requests as requests
+from showingpreviously.model import ChainArchiver, CinemaArchiverException, Chain, Cinema, Screen, Film, Showing
+
+CINEMAS_URL = 'https://vwc.odeon.co.uk/WSVistaWebClient/ocapi/v1/browsing/master-data/sites'
+FILMS_URL = 'https://vwc.odeon.co.uk/WSVistaWebClient/ocapi/v1/browsing/master-data/films'
+SHOWINGS_URL = 'https://vwc.odeon.co.uk/WSVistaWebClient/ocapi/v1/browsing/master-data/showtimes/business-date/{date}'
+
+JWT_TOKEN = None
+DAYS_AHEAD = 2
+CHAIN = Chain('Odeon')
+
+
+def get_jwt_token() -> str:
+    global JWT_TOKEN
+    if JWT_TOKEN is None:
+        driver = webdriver.Chrome()
```

Comment:
Might be worth running this headless, to avoid the annoying popup window.

Reply:
This seems to be complicated to do, especially cross-browser. Have you done this before?

Reply:
It seems very straightforward. I've got it working with Chrome ([these instructions](https://stackoverflow.com/questions/53657215/running-selenium-with-headless-chrome-webdriver)) and Firefox ([these instructions](https://www.tutorialspoint.com/how-to-make-firefox-headless-programmatically-in-selenium-with-python))